SPECS/net-snmp/net-snmp.spec

```diff
@@ -92,7 +109,20 @@ popd
 %{_libdir}/*.so
 %exclude %{_libdir}/perl5/perllocal.pod
 
+%files libs
+%doc COPYING README FAQ NEWS TODO
+%{_libdir}/libnetsnmp.so.*
```

Comment:
This file looks like it is included in both the base package and the libs subpackage. Can you exclude this library from the base package to avoid double-packaging?